biz_portal/apps/portal/templates/portal/business.html

```diff
@@ -1,5 +1,134 @@
 {% extends "base.html" %}
+{% load static %}
 
 {% block content %}
-  <p>{{ object.registered_name }}</p>
+
+  <!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="{% static 'biz-portal.bundle.css' %}">
+    <title>Business Directory</title>
+  </head>
```

Comment:
from DOCTYPE to here should be in base.html

Reply:
Done